project/main.py

```
@@ -1,22 +1,34 @@
 
 def run_server():
-    uvicorn.run(app, host="0.0.0.0", port=8000)
+    os.makedirs("logs", exist_ok=True)
+    log_config_file_path = get_project_root() / "config" / "logging.json"
 
+    with open(log_config_file_path) as f:
+        log_config = json.load(f)
+
+    logging.config.dictConfig(log_config)
+
+    uvicorn.run(app, host="0.0.0.0", port=8000, log_config=log_config)
```

Ruff (0.15.10)

[error] Possible binding to all interfaces

(S104)